connector/oidc/oidc.go

```diff
@@ -186,18 +189,18 @@ type oidcConnector struct {
 	preferredUsernameKey                  string
 	emailKey                              string
 	groupsKey                             string
+	forwardedLoginParams                  []string
 }
 
 func (c *oidcConnector) Close() error {
 	c.cancel()
 	return nil
 }
 
-func (c *oidcConnector) LoginURL(s connector.Scopes, callbackURL, state string) (string, error) {
+func (c *oidcConnector) LoginURL(s connector.Scopes, callbackURL, state string, forwardParams url.Values) (string, error) {
 	if c.redirectURI != callbackURL && !c.insecureSkipIssuerCallbackDomainCheck {
 		return "", fmt.Errorf("expected callback URL %q did not match the URL in the config %q", callbackURL, c.redirectURI)
 	}
-
```

Comment:
Nit: this is extremely superficial, but it might help get this PR approved if we left the space in (i.e. we want to match the surrounding code as much as possible—the Dex maintainers would be supporting whatever code we add, so ideally they should feel like they wrote it)

Reply:
good point!